### PR TITLE
AN-5237/ native transfers fix

### PR DESCRIPTION
**Pull request by forgxyz**
Primary change is that this removes the inner join to txs on native near `Transfer` actions.
1. When we migrated the transfers table from just native NEAR to include FTs and deployed the new ez table, columns like tx_signer and tx_fee were dropped. So, it is no longer needed and is burning extra compute.
2. The inner join is causing a significant gap. Details in slack.

Required deployment steps:
 - Delete * from transfers complete where type = native (instead of relying on incremental logic due to swap in source)
 - Generate this model new (~15 mins) + downstream transfers models
 - FR `silver__lockup_actions` which has a dependency on native NEAR transfers

### CODE REVIEW

File: models/silver/curated/token_transfers/native/silver__token_transfer_native.sql [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  WHERE
+    action_name = 'Transfer' 
+    
+    {% if var("MANUAL_FIX") %}
+      AND {{ partition_load_manual('no_buffer') }}
+    {% else %}
+{% if is_incremental() %}
+AND _modified_timestamp >= (
+  SELECT
+    MAX(_modified_timestamp)
```
Comment (eric-laurello) [inline models/silver/curated/token_transfers/native/silver__token_transfer_native.sql:40]: column doesnt exist in {{this}}

File: models/silver/curated/token_transfers/native/silver__token_transfer_native.yml [hunk 2]
```diff
@@ -36,32 +36,17 @@ models:
...
-          - not_null:
-              where: _inserted_timestamp <= current_timestamp - interval '1 hour'
+      - name: GAS_PRICE
+        description: "{{ doc('gas_price')}}"
+
+      - name: GAS_BURNT
+        description: "{{ doc('gas_burnt')}}"
+
+      - name: TOKENS_BURNT
+        description: "{{ doc('tokens_burnt')}}"
```
Comment (eric-laurello) [inline models/silver/curated/token_transfers/native/silver__token_transfer_native.yml:49]: this doc is missing

File: models/silver/curated/staking/silver__pool_events.sql [hunk 3]
```diff
@@ -25,6 +37,7 @@ FINAL AS (
         block_id,
         block_timestamp,
         receipt_object_id,
+        predecessor_id,
```
Comment (eric-laurello) [inline models/silver/curated/staking/silver__pool_events.sql:40]: just adding this in unrelated to the other changes?

File: models/silver/curated/token_transfers/native/silver__token_transfer_native.sql [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  predecessor_id,
+  receiver_id,
+  signer_id,
+  receipt_succeeded,
+  gas_price,
+  gas_burnt,
+  tokens_burnt,
+  _partition_by_block_number,
+  _inserted_timestamp,
+  {{ dbt_utils.generate_surrogate_key(
```
Comment (eric-laurello) [inline models/silver/curated/token_transfers/native/silver__token_transfer_native.sql:67]: assuming this got added in later as there's a ton of nulls but the logic looks fine.

File: models/silver/curated/staking/silver__pool_events.sql (hunk 3 above)
Comment (forgxyz) [reply models/silver/curated/staking/silver__pool_events.sql:40]: Yea, have to fr it and predecessor id is a very important col so figured I might as well add it during fr

Comment (forgxyz) [reply models/silver/curated/staking/silver__pool_events.sql:40]: Actually, this is not downstream of the transfers table. May not have meant to add this col at this time

**Review by eric-laurello — APPROVED** [review]

**Pull request merged**